app-metrics/prometheus-podman-exporter/prometheus-podman-exporter-1.4.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module systemd
+
+DESCRIPTION="Exporter for podman giving containers, pods, images, volumes & networks metrics"
+HOMEPAGE="https://github.com/containers/prometheus-podman-exporter"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/prometheus-podman-exporter.git"
+else
+	SRC_URI="https://github.com/containers/prometheus-podman-exporter/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+# Main package
+LICENSE="Apache-2.0"
+# Dependencies
+LICENSE+=" BSD-2 BSD ISC MIT MPL-2.0"
+SLOT="0"
+RESTRICT="test"
+RDEPEND="
+	>=app-containers/podman-4.0.0
+"
+DEPEND="${RDEPEND}"
+
+src_compile() {
+	export BUILDFLAGS=" -tags exclude_graphdriver_devicemapper"
+	has_version app-containers/podman[-btrfs] && BUILDFLAGS+=",exclude_graphdriver_btrfs,btrfs_noversion"
```

Comment:
I like this style, but this isn't truly a working solution. If you flip the `btrfs` flag on podman, nothing will happen for this package. Here you want to introduce a use flag and control it with `app-containers/podman[btrfs=]`.

Reply:
Done. But implemented with different logic. If btrfs use is enabled then force podman[btrfs]  else just depend on podman